backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5754820_sys_gh23380_Analysis_Report_Window_Update.sql

```diff
@@ -0,0 +1,161 @@
+-- Column: QM_Analysis_Report.M_Product_ID
+-- Column: QM_Analysis_Report.M_Product_ID
```

Comment:
maybe i am missing something obvious,
but how will be the QM_Analysis_Report.M_Product_ID populated.

same question for ASI

Reply:
Manualy in the window:

<img width="427" alt="image" src="https://github.com/user-attachments/assets/23fc55ac-8722-4514-a4da-60f1b3aa7f3e" />